deneme3.py

```diff
@@ -0,0 +1,102 @@
+# -*- coding: utf-8 -*-
+from py2neo import authenticate, Graph,Node
+
+import argparse
+
+parser = argparse.ArgumentParser()
+parser.add_argument('--user', type=str, default="neo4j", help='username for neo4j')
+parser.add_argument('--password', type=str, default="777", help='password of the user')
+
+args = parser.parse_args()
+
+authenticate("localhost:7474", args.user, args.password)
+graph = Graph()
+start=graph.begin()
+#g=Graph("bolt://localhost:7687", auth=("neo4j", "777"))
+
+#string="CREATE (Node:Actor{userid: "", fullname: ""})"
+
+path=r"C:\Users\ASUS\Documents\staj\data\FILMS.txt"
+f=open(path,"r")
+actor_list=[]
+
+text_lines=f.readlines()
+i=0
+print(len(text_lines))
+while i<len(text_lines):
+    line=text_lines[i].split(',')
+    i+=1
+    print(len(line))
+    director=line[8]

```

Comment:
Ayrı ayrı değişken tanımlamak yerine Python dictionary kullanabiliriz. (Search key: Python dictionary)
Örneğin:
```
line = text_lines[i].split(' % ')
actors = line[3].split(', ')
movie[movie_id] = {          
    'id'       : line[0],
    'title'    : line[1],
    'year'     : line[2],
    'actors'   : actors,
    'genre'    : line[4],
    'director' : line[5],
    'rating'   : line[6]
}
```